src/Microsoft.AspNetCore.DataProtection/DataProtectionUtilityExtensions.cs

```diff
@@ -37,10 +36,6 @@ public static string GetApplicationUniqueIdentifier(this IServiceProvider servic
             if (services != null)
             {
                 discriminator = services.GetService<IApplicationDiscriminator>()?.Discriminator;
```

Comment:
This service won't be automatically registered when using DP without the hosting stack so `GetRequiredService` will throw an exception an prevent the app from loading/working correctly.